molli/external/rdkit.py

```diff
@@ -0,0 +1,1095 @@
+        
+        else:
+            if raise_kekulize:
+                raise RDKitKekulizationException(f'{m.name} is failing to kekulize with {ext} parsing')
+            else:
+                rdmol.SetProp(problems[0], "1")
+
+    return rdmol
+
+def _rd_mol2(m:ml.Molecule, out:str, remove_hs:bool, ext:str, raise_kekulize=True) -> PropertyMol:
+    '''Loads Molecule into RDKit with RDKit's MolFromMol2Block. Will process potential problems with kekulization.
+
+    Parameters
+    ----------
+    m : ml.Molecule
+        Molli Molecule
+    out : str
+        Mol2Block 
+    remove_hs : bool
+        Will remove hydrogens if specified
+    ext : str
+        Extension format used
+    raise_kekulize : bool
+        Can be used to prevent instantiation if Molecule fails to kekulize
+
+    Returns
+    -------
+    PropertyMol
+        New RDKit Mol Object
+    '''
+
+    rdmol = PropertyMol(
+        Chem.MolFromMol2Block(out, removeHs=remove_hs)
+    )
+    if rdmol is None:
+        rdmol = Chem.MolFromMol2Block(out, removeHs=remove_hs, sanitize=False)
+
+    rdmol = _rd_problems(m, rdmol, ext, raise_kekulize)
+
+    return rdmol
+
+def _rd_xyz(m:ml.Molecule, out:str, remove_hs:bool, ext:str, raise_kekulize=True) -> PropertyMol:
+    '''Loads Molecule into RDKit with RDKit's MolFromXYZBlock. Will process potential problems with kekulization.
+
+    Parameters
+    ----------
+    m : ml.Molecule
+        Molli Molecule
+    out : str
+        XYZBlock 
+    remove_hs : bool
+        Will remove hydrogens if specified
+    ext : str
+        Extension format used
+    raise_kekulize : bool
+        Can be used to prevent instantiation if Molecule fails to kekulize
+
+    Returns
+    -------
+    PropertyMol
+        New RDKit Mol Object
+    '''
+
+    rdmol = PropertyMol(
+        Chem.MolFromXYZBlock(out)
+    )
+    if remove_hs:
+        Chem.RemoveHs(rdmol)
+
+    rdmol = _rd_problems(m, rdmol, ext, raise_kekulize)
+    return rdmol
+
+def _rd_mol(m:ml.Molecule, out:str, remove_hs:bool, ext:str, raise_kekulize=True) -> PropertyMol:
+    '''Loads Molecule into RDKit with RDKit's MolFromMolBlock. Will process potential problems with kekulization.
+
+    Parameters
+    ----------
+    m : ml.Molecule
+        Molli Molecule
+    out : str
+        MolBlock 
+    remove_hs : bool
+        Will remove hydrogens if specified
+    ext : str
+        Extension format used
+    raise_kekulize : bool
+        Can be used to prevent instantiation if Molecule fails to kekulize
+
+    Returns
+    -------
+    PropertyMol
+        New RDKit Mol Object
+    '''
+
+    rdmol = PropertyMol(
+        Chem.MolFromMolBlock(out, removeHs=remove_hs)
+    )
+    if rdmol is None:
+        rdmol = Chem.MolFromMolBlock(out, removeHs=remove_hs, sanitize=False)
+
+    rdmol = _rd_problems(m, rdmol, ext, raise_kekulize)
+    return rdmol
+
+def _rd_pdb(m:ml.Molecule, out:str, remove_hs:bool, ext:str, raise_kekulize=True) -> PropertyMol:
```

Comment:
just FYI if could be coded way easier if you just passed the loader as a function argument. For instance:
```
def _to_rd(thing, converter, *args, **kwargs):
    return converter(thing, *args, **kwargs)
```
And then you call it as `_to_rd(thing, Chem.MolFromMolBlock)`.